src/App.tsx

```diff
@@ -94,9 +119,21 @@ function App() {
 			{!schemaStore.isLoading ? (
 				<div className={classes.content}>
 					<Boxes />
-					<Config />
-					<Metrics />
-					<Links />
+					{
+						schemaStore.selectedDictionary
```

Comment:
I think we can now create two component for each view DictionaryLayout, BoxLayout
DictionaryLayout will have another components not just editor

There is another problem that after selecting dictionary there is no way to switch back other than click button, even if i click box nothing happens